doc/articles/migrating-to-uno-6.md

```diff
@@ -94,8 +94,6 @@ To upgrade:
     ```csharp
     public static int Main(string[] args)
     {
-        App.InitializeLogging();
```

Comment:
This is required for the blank template. We should make it conditional to another call being present or at least explain how to check if it's needed here.

Reply:
It's not relevant to the changes we are specifying here that are required when migrating an existing app to Uno 6 Skia

We could just replace with `// Code omitted for brevity`